modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,340 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer (ALB) Operator
+Users that wish to install the AWS Load Balancer (ALB) Operator can do so as long as the following prerequisites are met.
+
+.Prerequisites
+
+* A multi-AZ ROSA cluster deployed with STS and created using customer-managed Subnets
+* Access to modify the VPC and Subnets of the created {product-title} cluster
+// * AWS account credentials
+* Configured {product-title} (ROSA) CLI
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+
+[NOTE]
+====
+When installing for use in an AWS Local Zone, the Local Zone is enabled for the account and the ALB Operator is available within the Local Zone/region.
+====
+
+
+.Procedure
+
+. Specify the cluster infrastructure ID and the cluster OpenID Connect (OIDC) DNS using the following commands:
+.. Specify the {product-title} (ROSA) cluster INFRA ID:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'Infra ID' .. OR   $oc get infrastructure cluster -o json | jq -r '.status.infrastructureName' ----
+----
+
+.. Specify the {product-title} (ROSA) cluster OIDC DNS:
++
+--
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'OIDC'
+----
+
+Save the output from both commands. You will use this information in future steps within this procedure.
+
+--
+. Create and manage AWS Identity and Access Management (IAM) roles and policies:
++
+.. Create the AWS IAM policy required for the the AWS Load Balancer Controller (ALBC) and the AWS Load Balancer (ALB) Operator with the following content:
++
+[source,json]
```

Comment:
Should we have an expand/collapse option for this to make more readable/concise the rest of the instructions?
Should we use the URL of the source to make sure there are not typos and drift from source?

Reply:
> Should we have an expand/collapse option for this to make more readable/concise the rest of the instructions? 
Maybe (probably) so. Will have to do some restructuring overall. Will continue to work on seeing about restructuring.  
Should we use the URL of the source to make sure there are not typos and drift from source? Was curious to see what you would say about this :). Yes, I will include URL instead of inserted JSON info. Also will shorten the docs by doing so.